metadata/demographic/census_blocks_2020.md

```diff
@@ -0,0 +1,90 @@
+
+### What is the purpose of the dataset
+
+Geography is a fundamental aspect of the Census, providing the framework for the once-a-decade count of population and housing. Read about some of the key geography changes in the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).
+
+### What does the dataset represent
+
+Each Census block in this dataset is given a unique four-digit Census Block Number. For more information on how to interpret the Census Block Number, refer to this article from the [US Census Bureau](https://www.census.gov/newsroom/blogs/random-samplings/2011/07/what-are-census-blocks.html). While census blocks include information on major racial groups, housing, and population, it is important to note that census blocks are delineated by physical and municipal boundaries, not by population size. Thus, a larger block does not denote a larger population size. In fact, many census blocks contain a population number of zero (such as in the case of water-only areas).
+
+### How was the dataset created
+
+Utah's 2020 census geographic boundaries became available in February 2021 and the demographic redistricting legacy files (the 2020 Census Redistricting P.L. 94-171 Summary Files) became available on August 12, 2021. The Census Bureau will release a more user-friendly version of the 2020 Census Redistricting P.L. 94-171 Summary Files on September 30, 2021. In the meantime, UGRC, with collaboration from the Analytics Group at [WFRC](https://wfrc.org/), has worked with the August 12 release and has created ready-to-use GIS data in several formats that improve the user-friendliness of the 2020 redistricting data. Read more about these offerings in the blog post, 2020 Census Redistricting Data in Easy to Use GIS Formats
+
+### How reliable and accurate is the dataset
+
+The 2020 data available on this page was downloaded from the [2020 Census Redistricting Data (P.L. 94-171)](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2). All 2020 census geographies reflect boundaries as of January 1, 2020. Additional GIS files and support products are available on the [census redistricting data](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2).
+
+These datasets were downloaded from the 2020 Census Redistricting Data (P.L. 94-171) Page. All 2020 Census boundaries are current as of January 1, 2020. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, AGRC will append the demographic data to the existing 2020 geographies found on this page.
```

Comment:
**suggestion**: make these singular

```suggestion
This dataset was downloaded from the 2020 Census Redistricting Data (P.L. 94-171) Page. All 2020 Census Block boundaries are current as of January 1, 2020. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, UGRC will append the demographic data to the existing 2020 geographies found on this page.
```

**question**: has the appending already happend? Should it be re explained?
**question**: should the current as of be displayed here again?

Reply:
I believe the appending has already occurred. I have adjusted this paragraph accordingly, the changes will be reflected in a new pull request. Thank you for your feedback!